docs/collect_examples.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020  The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier:	ISC
+
+import os
+from shutil import copy, copytree, ignore_patterns
+
+from make_xml_for_docs import make_xml_for_docs
+
+# =============================================================================
+
+
+README_NAME = "README.rst"
+TEMPLATE_NAME = "examples.rst.template"
+INDEX_NAME = "examples.rst"
+
+
+def process_verilogs(src_dir, dst_dir):
```

Comment:
Verilogs is not a real word :-)

```suggestion
def process_verilog_files(src_dir, dst_dir):
```
or
```suggestion
def process_verilog_examples(src_dir, dst_dir):
```
or
```suggestion
def process_verilog_tests(src_dir, dst_dir):
```

Reply:
True... Fixed that.